src/AcceptanceTests/When_operating_with_least_privilege.cs

```diff
@@ -0,0 +1,114 @@
+namespace NServiceBus.Transport.AzureServiceBus.AcceptanceTests
+{
+    using System;
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using Features;
+    using NServiceBus.AcceptanceTests;
+    using NServiceBus.AcceptanceTests.EndpointTemplates;
+    using NServiceBus.Configuration.AdvancedExtensibility;
+    using NUnit.Framework;
+
+    public class When_operating_with_least_privilege : NServiceBusAcceptanceTest
+    {
+        const string DedicatedTopic = "bundle-no-manage-rights";
+
+        [Test]
+        public async Task Should_work()
```

Comment:
nitpick, but can we clarify the test naming somehow to be more clear that this is "running in least privilege mode where all required resources already exist? And that this requires auto-subscribe to be disabled? e.g. something like this:

```suggestion
        public async Task Should_allow_message_operations_on_existing_resources
```

Reply:
Yeah sorry that was just a draft name